### ISSUE TITLE
How to reconcile image-heavy pages with text-heavy descriptions to allow text search

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazi-documentation#1]
Some pages have a heavy amount of images describing the issue.  Some, like this one:

`admin-docs/how-to-build-data-visualisation-components.md`

Hold almost EVERYTHING in images.  Text made into images in order to color code, create sections, arrows, etc.

While this is very useful for instructional purposes, it impedes the text search to find anything within that page.  How do we reconcile those two scenarios?

**Comment by fnocetti** [comment]
Agree, we'll need to find a strategy to write that page using MD tools and not images. Images will also make it more complex to translate it.

**Comment by nwyu** [comment]
Agree completely with everything said. This will be rewritten and redesigned in the next 1-2 months.
